prime-router/metadata/fhir_mapping/hl7/segments/MSH/Provenance-Transformation.yml

```diff
@@ -0,0 +1,57 @@
+resourceType: Provenance
+id:
+  type: STRING
+  valueOf: "GeneralUtils.generateResourceId()"
+  expressionType: JEXL
+
+target:
+  valueOf: datatype/Reference
+  generateList: true
+  expressionType: resource
+  specs: $Bundle
+
+activity:
+  expressionType: nested
+  expressionsMap:
+    coding:
+      expressionType: nested
+      expressionsMap:
+        code:
+          type: STRING
+          valueOf: "v2-FHIR transformation"
+
+recorded:
+  type: STRING
+  valueOf: "GeneralUtils.getCurrentDateTimeUsingZoneId(ZONEID)"
+  expressionType: JEXL
+
+policy:
+  type: SYSTEM_URL
+  valueOf: "http://hl7.org/fhir/uv/v2mappings/message-oru-r01-to-bundle"
+
+agent:
+  expressionType: nested
+  generateList: true
+  expressionsMap:
+    type:
+      expressionType: nested
+      expressionsMap:
+        coding:
+          expressionType: nested
+          expressionsMap:
+            code:
+              type: STRING
+              valueOf: "assembler"
+            system:
+              type: SYSTEM_URL
+              valueOf: "http://terminology.hl7.org/CodeSystem/provenance-participant-type"
+    who:
+      valueOf: segments/MSH/datatypes/HD/ReportStreamOrganization
+      expressionType: reference
+
+entity:
```

Comment:
I'm wondering if we should leave this off since we're not populating the document reference

Reply:
Hmm, yea maybe, the same would probably apply to Provenance-Source, and then I'd need to add an extension to be able to reliably distinguish between the different provenance types. That might be a good idea anyway, the existing fhir path to make that distinction is pretty clunky. I'll work through that.